core/bin/external_node/src/main.rs

```diff
@@ -704,6 +704,21 @@ struct Cli {
     /// Comma-separated list of components to launch.
     #[arg(long, default_value = "all")]
     components: ComponentsToRun,
+    /// Path to the yaml config. If set, it will be used instead of env vars.
+    #[arg(
+        long,
+        requires = "secrets_path",
+        requires = "external_node_config_path"
+    )]
+    config_path: Option<std::path::PathBuf>,
+    /// Path to the yaml with secrets. If set, it will be used instead of env vars.
+    #[arg(long, requires = "config_path", requires = "external_node_config_path")]
+    secrets_path: Option<std::path::PathBuf>,
+    /// Path to the yaml with genesis. If set, it will be used instead of env vars.
+    #[arg(long, requires = "config_path", requires = "secrets_path")]
+    external_node_config_path: Option<std::path::PathBuf>,
+    /// Path to the yaml with consensus.
+    consensus_path: Option<std::path::PathBuf>,
```

Comment:
Just to discuss: Would it make sense to specify a path to a directory that would contain all these config files with predefined names? This would allow to change the set of these files in the future w/o changing command-line args here. Also, this approach (configuration by convention) is commonly used in various frameworks.

Reply:
I was thinking about it and probably it's not that bad, as i thought before, i can touch it later